init.sh

```diff
@@ -36,10 +36,10 @@ if [[ $1 == "pending" ]]; then
 fi
 
 # Allocate genesis accounts (cosmos formatted addresses)
-ethermintd add-genesis-account $KEY 100000000000000000000aphoton --keyring-backend test
+ethermintd add-genesis-account $KEY 1000000000000000000aphoton,1000000000000000000stake --keyring-backend test
 
 # Sign genesis transaction
-ethermintd gentx $KEY --amount=1000000000000000000aphoton --keyring-backend test --chain-id $CHAINID
+ethermintd gentx $KEY 1000000000000000000stake --keyring-backend test --chain-id $CHAINID
```

Comment:
the denom here should be `aphoton` i believe. if it works with both, we may need to investigate that issue

Reply:
yeah you are right, amount is still needed here, it just that we need stake amount as well otherwise validator set would be empty